samples/customchecks/monitoring3rdparty/Version_3_3/3rdPartySystem/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net45" />
```

Comment:
How would that confuse the user?

And how does lightness (in terms of packages) correlate with confusion?



*From:* Simon Cropp [mailto:notifications@github.com]
*Sent:* Dienstag, 3. März 2015 23:39
*To:* Particular/docs.particular.net
*Cc:* Daniel Marbach
*Subject:* Re: [docs.particular.net] Monitoring 3rd party systems with
custom checks (#565)



In
samples/customchecks/monitoring3rdparty/Version_3_3/3rdPartySystem/packages.config
<https://github.com/Particular/docs.particular.net/pull/565#discussion_r25734864>
:

> @@ -0,0 +1,7 @@

> +﻿<?xml version="1.0" encoding="utf-8"?>

> +<packages>

> +  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net45" />

but you are not trying to demonstrate web api so familiarity with it is a
not an issue. and in this case we are trying to simulate an external web
service that they dont own so we should choose the lightest weight approach
so as to not confuse the user

—
Reply to this email directly or view it on GitHub
<https://github.com/Particular/docs.particular.net/pull/565/files#r25734864>
.